manifests/sysctl.pp

```diff
@@ -251,14 +251,14 @@
   }
   else {
     $_disable_ipv6 = $ipv6 ? { true => 0, false => 1 }
-    sysctl { 'net.ipv6.conf.all.disable_ipv6': value => $_disable_ipv6 }
   }
 
-  if $facts['ipv6_enabled'] and ( $_disable_ipv6 == 0 ) {
+  unless $facts['simplib_sysctl']['net.ipv6.conf.all.disable_ipv6'] =~ Undef {
```

Comment:
Can you change this to a `dig` on the `facts` hash in case `simplib_sysctl` isn't present for some reason?

You can also drop the `=~ Undef` since the `unless` will work properly